src/test/java/net/sf/jabref/logic/l10n/LocalizationConsistencyTest.java

```diff
@@ -156,7 +156,7 @@ public void findObsoleteLocalizationKeys() throws IOException {
             System.out.println();
             System.out.println("1. CAREFULLY CHECK IF THE KEY IS REALLY NOT USED ANYMORE");
             System.out.println("2. REMOVE THESE FROM THE ENGLISH LANGUAGE FILE");
-            System.out.println("3. EXECUTE gradlew -b localization.gradle generateMissingTranslationKeys TO");
+            System.out.println("3. EXECUTE gradlew update");
```

Comment:
Nope, `localization.gradle` is imported in the `build.gradle` file, thus gradle knows the tasks form `localization.gradle` too.
Just try it out.